src/on-premises/install/quickstart/config.md

```diff
@@ -0,0 +1,329 @@
+|----------------------------|------------------------------------|-------------------------------------------------------------|
+| bimdata_path               | "/opt/bimdata"                     | Where we will install our needed files on the servers.      |
+| bimdata_docker_volume_path | `"{{ bimdata_path }}/data"`        | Where will your data will be store on the servers.          |
+| bimdata_dockerfiles_path   | `"{{ bimdata_path }}/dockerfiles"` | Where we store the dockerfiles use to start the containers. |
+:::
+
+Object storage (Swift):
+
+::: v-pre
+| Variables                    | Default value                    | Description                                         |
+|------------------------------|----------------------------------|-----------------------------------------------------|
+| swift_enabled                | false                            | Enable the swift storage or not.                    |
+| swift_auth_url               | ""                               | The URL of the auth server.                         |
+| swift_tenant_id              | ""                               | The tenant/project id to use when authenticating.   |
+| swift_tenant_name            | ""                               | The tenant/project name to use when authenticating. |
+| swift_username               | ""                               | The username to use to authenticate.                |
+| swift_password               | "{{ vault_swift_password }}"     | The password/key to use to authenticate.            |
+| swift_temp_url_key           | "{{ vault_swift_temp_url_key }}" | The temporary URL key ([see openstack documentation](https://docs.openstack.org/kilo/config-reference/content/object-storage-tempurl.html))      |
+| swift_api_container_name     | ""                               | The container in which to store the API files.      |
+| swift_connect_container_name | ""                               | The container in which to store the Connect files.  |
+:::
+
+### Applications configuration
+::: v-pre
+| Variables                           | Default value                                                              | Description                                                 |
+|-------------------------------------|----------------------------------------------------------------------------|-------------------------------------------------------------|
+| api_secret_key                      | `"{{ vault_api_secret_key }}"`                                             | You should not change this.                                 |
+||||
+| connect_secret_key                  | `"{{ vault_connect_secret_key }}"`                                         | You should not change this.                                 |
+| connect_client_id                   | `"{{ 'connect_client_id' \| to_uuid(namespace=uuid_namespace) }}"`         | You should not change this.                                 |
```

Comment:
It's the best ansible native solution I could think of to have a more or less unique UUID even between different customers.
It take the string 'connect_client_id' and transform it in a namespaced UUIDv5. So the UUID is always the same between each playbook run (UUIDv5 are deterministic) but we use the namespace feature to make them different for each customer. The namespace is define by `uuid_namespace: "{{ app_dns_domain | to_uuid }}"` so it's also always the same UUID, but it should be different for each customer because it hash their DNS domain.
I though it was a good idea to try to provide safe real UUIDs by default and not always the same (I saw situations when companies try to merge where it add unnecessary complexity in the migration when a product have "non unique uuid" ^^).

Reply:
Do we want to tell about this technical stuff in the doc? Could we just say that the default value is a deterministic uuid ?